client/src/components/Nav/Nav.tsx

```diff
@@ -76,16 +76,16 @@ const Nav = ({ navVisible, setNavVisible }) => {
   }, [tags]);
   const { containerRef, moveToTop } = useNavScrolling<ConversationListResponse>({
     setShowLoading,
-    hasNextPage: searchQuery ? searchQueryRes?.hasNextPage : hasNextPage,
-    fetchNextPage: searchQuery ? searchQueryRes?.fetchNextPage : fetchNextPage,
-    isFetchingNextPage: searchQuery ? searchQueryRes?.isFetchingNextPage : isFetchingNextPage,
+    hasNextPage: searchQuery ? searchQueryRes.hasNextPage : hasNextPage,
```

Comment:
Please remove these changes, as `searchQuery` is not typed correctly (an issue before this PR) so following the linter here will result in error.